style.css

```
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  scroll-behavior: smooth;
+}
+
+.logo {
+  height: 21px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 21px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #4c59d1;
+  margin-right: auto;
+  margin-left: 24px;
+}
+
+.icon div {
+  border: 2px solid #4c59d1;
+  width: 22px;
+  margin: 3px;
+  border-radius: 15%;
+}
+
+.topnav {
+  display: flex;
+  align-items: center;
+  padding-right: 36px;
+  height: 48px;
+}
+
+.headline {
+  padding: 114px 24px;
+  background-image: url(./images/background1.png);
```

- [ ] Here's the typo: 🤓 

```suggestion
  background-image: url(./Images/background1.png);
```